tmt/tests/bootc-install-provision.sh

```diff
@@ -19,7 +19,27 @@ case "$ID" in
 esac
 
 if [ "$TMT_REBOOT_COUNT" -eq 0 ]; then
+    # Fedora CI: https://github.com/fedora-ci/dist-git-pipeline/blob/master/Jenkinsfile#L145
```

Comment:
This is something I keep getting hung up on. So AIUI today, Testing Farm basically does not handle the job of injecting (or building) the artifact-under-test, it's the reponsibility of a different tool. And commonly today, that different tool is [Packit](https://packit.dev/) (which is very RPM oriented).

Would it be accurate then to say this is basically doing something similar to what Packit is doing?

Is there hookup/alignment between the fedora-ci/osci and packit or do they just overlap in this sense?

Reply:
AFAIK, the fedora ci and osci don't use Packit yet.